spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+describe 'POST api/v1/targets/', type: :request do
+  let(:user) { create(:user) }
+  let(:topic) { create(:topic) }
+  let(:failed_response) { 400 }
```

Comment:
I'm not sure but did you try to just call :bad_request the same way you do with :success 🤔

Reply:
Can;t use it because of the expect change blocks 😬